reference/ingestion-tools/index.md

```diff
@@ -1,16 +1,17 @@
 # Ingestion tools
 
-% TO-DO: Add links to "What are Ingestion tools?"%
-
 This section contains reference information for ingestion tools, including:
 
 * Fleet and agent
 * APM
 * Beats
-* Enrich processor reference
+* Processor reference
 * Logstash
+* Logstash Plugins
+* Logstash Versioned Plugin Reference
 * Elastic Serverless forwarder for AWS
 * Search connectors
-* ES Hadoop
+* ES for Apache Hadoop
+* Elastic in integrations
 
-This document is intended for programmers who want to interact with the ingestion tools and doesn't contain information about the API libraries.
\ No newline at end of file
+For more information about the ingestion tools, check out the [Ingest tools overview](docs-content://manage-data/ingest/tools.md) documentation.
```

Comment:
Would it help to give people an idea of what's different about the non-reference info?
Hopefully the solution to the build error is to just drop the repo info:
```suggestion
For introductory content and broader overviews, check out the [Ingest tools overview](/manage-data/ingest/tools.md) documentation.
```